README.md

```diff
@@ -0,0 +1,88 @@
+# Terraform Transit Gateway "satellite" module
+
+We are following the hub-spoke(s) (aka [star network][1]) network topology
+model.
+
+This Terraform module aims to handle the AWS resources required by a so-called
+"satellite" node.
+
+It is important to mention that the invocation of this module is going to
+actually make cross-account changes. That is, we need to adjust the
+configuration on the hub side (mainly routing).
+
+This module assumes that its pair module was used:
+[terraform-aws-transit-gateway-hub][2] to handle the actual TGW.
+
+Check out the [examples][3] in the aforementioned module.
+
+## Supported resources
+
+At the moment, this module only supports attaching _VPCs_ to the TGW.
+Support for VPN tunnels will be added soon.
+
+## Assumptions
+
+### Credentials
+
+The module starts from the assumption that the `aws_login_profile` allows the
+user to assume the necessary IAM roles, as required, to make the necessary
+changes (and in the case of the `satellite` module, cross-account).
+
+See [this example][4] to first make sure that the credentials you want to use
+allow for cross-account actions.
+
+You can read more about how Terraform handles this [here][5].
+
+Obviously, all the [supported authentication][6] methods can also be used.
+
+### Routing
+
+For creating the network routes on the satellite side, the module expects to
+find the keyword "_private_" in the name of the subnets, so that it may collect
+their IDs and those of their associated routing tables. 
+Check the `subnet_name_keyword_selector` variable if you want to change this.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws.hub | n/a |
+| aws.satellite | n/a |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| aws\_account\_id\_hub | AWS account number containing the TGW hub | `string` | n/a | yes |
+| aws\_login\_profile | Name of the AWS login profile as seen under ~/.aws/config used for assuming cross-account roles | `any` | n/a | yes |
+| ram\_resource\_association\_id | Identifier of the Resource Access Manager Resource Association | `string` | n/a | yes |
+| role\_to\_assume\_hub | IAM role name to assume in the AWS account containing the TGW hub (eg. ASSUME-ENG-OPS) | `string` | n/a | yes |
```

Comment:
To make it clean let's rename it to `ASSUME-ROLE-HUB`

Reply:
Thanks for the reminder! I remember seeing that later on but then forgot to correct it.